src/components/FlawTrackers.vue

```diff
@@ -0,0 +1,460 @@
+<script setup lang="ts">
+import { computed, ref, type Ref } from 'vue';
+import { formatDate } from '@/utils/helpers';
+import type { ZodAffectType, ZodTrackerType } from '@/types/zodAffect';
+import { ascend, descend, sortWith } from 'ramda';
+import TrackerManager from '@/components/TrackerManager.vue';
+import { useSettingsStore } from '@/stores/SettingsStore';
+import { usePagination } from '@/composables/usePagination';
+
+type TrackerWithModule = ZodTrackerType & { ps_module: string };
+
+const props = defineProps<{
+  flawId: string;
+  displayedTrackers: TrackerWithModule[];
+  affectsNotBeingDeleted: ZodAffectType[];
+  allTrackersCount: number;
+}>();
+
+const emit = defineEmits<{
+  'file-tracker': [value: object];
+  'affects:refresh': [];
+}>();
+
+const settingsStore = useSettingsStore();
+const settings = ref(settingsStore.settings);
+
+const showTrackerManager = ref(false);
+
+const hasTrackers = computed(() => props.allTrackersCount > 0);
+
+// Sorting
+const sortedTrackers = computed(() =>
+  sortTrackers(filteredTrackers.value)
+);
+
+type sortKeys = keyof Pick<ZodTrackerType,
+  'created_dt' | 'updated_dt'
+>;
+
+const sortKey = ref<sortKeys>('created_dt');
+const sortOrder = ref(ascend);
+
+const setSort = (key: sortKeys) => {
+  if (sortKey.value === key) {
+    sortOrder.value = sortOrder.value === ascend ? descend : ascend;
+  } else {
+    sortKey.value = key;
+    sortOrder.value = ascend;
+  }
+};
+
+function sortTrackers(trackers: TrackerWithModule[]): TrackerWithModule[] {
+  const customSortKey = sortKey.value;
+  const order = sortOrder.value;
+
+  const customSortFn = (affect: TrackerWithModule) => {
+    return affect[customSortKey] || 0;
+  };
+
+  const comparator = [order<TrackerWithModule>(customSortFn)];
+
+  return sortWith([
+    ...comparator
+  ])(trackers);
```

Comment:
Since comparator is already an array and is the only item I think can be changed 
```suggestion
  const comparator = [order<TrackerWithModule>(customSortFn)];

  return sortWith(comparator)(trackers);
```